### PR TITLE
Fixed wrong spacing for Sort menu in file list

### PR DESCRIPTION
**Pull request by hosy**
## Description
StackView has only two elements, spacing on the right side is to less and sort menu is not centered in view. Added the width value of the left button to the right anchor of the stack view to get the sort menu centered.

## Related Issue
#278 

## Motivation and Context
UI fix, because I can not unsee, that element is not centered in view.

## How Has This Been Tested?
Open file list on iPhone. Please check that everything is fine also in landscape mode.
Test also on iPad, if UISegmentedControl is centered.

## Screenshots (if appropriate):
simulator screen shot - iphone 8 plus - 2019-02-21 at 11 23 50

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: ownCloud/Client/SortBar.swift [hunk 1]
```diff
@@ -163,7 +163,7 @@ class SortBar: UIView, Themeable {
 			stackView.topAnchor.constraint(equalTo: topAnchor),
 			stackView.bottomAnchor.constraint(equalTo: bottomAnchor),
 			stackView.leftAnchor.constraint(equalTo: safeAreaLayoutGuide.leftAnchor, constant: 20),
-			stackView.rightAnchor.constraint(equalTo: safeAreaLayoutGuide.rightAnchor, constant: -20)
+			stackView.rightAnchor.constraint(equalTo: safeAreaLayoutGuide.rightAnchor, constant: -50)
```
Comment (mneuwert) [inline ownCloud/Client/SortBar.swift]: @hosy I am not a friend of magic numbers spread over the code. Would rather prefer using let constants. But may this is something we could do later.

Comment (michaelstingl) [reply ownCloud/Client/SortBar.swift]: > magic numbers

👎

Comment (hosy) [reply ownCloud/Client/SortBar.swift]: That's correct. There a many magic numbers to fix in this implementation. This was just a quick fix but I can do this now.

Comment (mneuwert) [reply ownCloud/Client/SortBar.swift]: @hosy @michaelstingl I think this is not only UI class which uses magic numbers. We could create a separate task to move spacing, margin values into a separate file defining these constants.

**Comment by michaelstingl** [comment]
@hosy Portrait and Landscape? With notch and without notch?

**Comment by hosy** [comment]
@michaelstingl yes, tested in all variants.

**Comment by hosy** [comment]
@michaelstingl safeAreaLayoutGuide is used for left and right padding (top and bottom is not relevant for this view)

**Comment by jesmrec** [comment]
Approved.

**Pull request merged**